src/menus/PlayerSelect.java

```diff
@@ -0,0 +1,203 @@
+package menus;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import utils.Gamepad;
+import DLibX.*;
+
+import java.awt.Color;
+import java.awt.Font;
+
+public class PlayerSelect extends Menu {
+
+    private ArrayList<Gamepad> players = new ArrayList<Gamepad>(Arrays.asList(null, null, null, null));
```

Comment:
Since this is going to have a max of 4 people always, this could be a simple array, and could reduce the number of times we repeat `.get(x)`